frontend/src/pages/Alerts/AlertConfigurationCard/AlertConfigurationCard.tsx

```diff
@@ -782,6 +786,7 @@ export const AlertConfigurationCard = ({
 											mode="multiple"
 											onSearch={(value) => {
 												setSearchQuery(value)
+												syncSlack()
```

Comment:
As discussed in the demo, I think a better approach here could be to kick this off once, maybe in an `onFocus` or even when the component is rendered.